### PR TITLE
Some Updates

### PR DESCRIPTION
**Pull request by derekcollison**
Made some updates to the lib.

Updated go vet and go fmt changes.

Remove ioutil usage.

Updated travis to go 1.19 and 1.18 versions for tests.

Updated go modules version to 1.18.

Signed-off-by: Derek Collison <derek@nats.io>

### CODE REVIEW

File: v2/go.mod [hunk 1]
```diff
@@ -1,5 +1,10 @@
 module github.com/nats-io/jwt/v2
 
-go 1.16
+go 1.18
 
-require github.com/nats-io/nkeys v0.3.0
+require (
+	github.com/nats-io/jwt v1.2.2
```
Comment (wallyqs) [inline v2/go.mod:6]: I think this will make vulnerability checks fail on nats-io/jwt v2 because v1 interpreted as a dependency of v2 not just for tests.

Comment (derekcollison) [reply v2/go.mod:6]: ok will revisit.

Comment (derekcollison) [reply v2/go.mod]: reverted.

**Review by wallyqs — APPROVED** [review]
LGTM

**Pull request merged**